cmd/ftl/cmd_schema_import.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/TBD54566975/ftl/backend/common/exec"
+	"github.com/TBD54566975/ftl/backend/common/log"
+	"github.com/TBD54566975/ftl/backend/common/slices"
+	"github.com/tmc/langchaingo/llms"
+	"github.com/tmc/langchaingo/llms/ollama"
+)
+
+type schemaImportCmd struct {
+	OllamaPort int             `help:"Port to use for ollama." default:"11434"`
+	Go         importGoCmd     `cmd:"" help:"Import types for an FTL Go module."`
+	Kotlin     importKotlinCmd `cmd:"" help:"Import types for an FTL Kotlin module."`
+}
+
+type importGoCmd struct {
+	Dir string `arg:"" required:"" help:"Directory to import from."`
+}
+
+type importKotlinCmd struct {
+	Dir string `arg:"" required:"" help:"Directory to import from."`
+}
+
+const ollamaContainerName = "ftl-ollama-1"
+const ollamaVolume = "ollama:/root/.ollama"
+const ollamaModel = "llama2"
+
+func (i *importKotlinCmd) getPrompt() (string, error) {
+	input, err := createInputString(i.Dir)
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("I will provide you with an input string containing the contents of a file which must be translated into Kotlin. "+
```

Comment:
These strings look identical? If they are, parameterise the language. You can use `%[1]s` to reuse a parameter index with Sprintf.

Reply:
they have some overlap but aren't identical

let's take a look at the prompts as a team tomorrow - if the results we land on are same except for language i can put this change in for sure